main/src/main/scala/sbt/Defaults.scala

```diff
@@ -1999,6 +1999,7 @@ object Classpaths {
       ConflictWarning(conflictWarning.value, report, log)
       report
     },
+    evictionWarningOptions in update := (evictionWarningOptions in GlobalScope).value,
```

Comment:
RHS should be `evictionWarningOptions.value,`

Reply:
Done in https://github.com/sbt/sbt/pull/3947/commits/dc4f93a7c3ecf2e2491aceaaecdbcec8fc8f7df9